CDCgov/NEDSS-Modernization
# Cnft2 1469 manage tabs drag drop reorder

## Description

Here we implement the drag drop reorder functionality to ManageTabs.
The DragDropContext was refactored some to accomodate Tabs but also improve functionality in the Section Reorder Modal.
Some ManageTabs design updates were added to this branch.

## Tickets

* https://cdc-nbs.atlassian.net/browse/CNFT2-1469

## Steps to verify

1. Select a Page from the Page Library
2. Click the `Manage Tabs` button in the Tabs navbar.
3. Drag a tab by the handle and move into a new position.
4. Tab order should update, as well as the Tab order on the Page
